packages/apollo/src/components/UpdateChannelMspModal/UpdateChannelMspModal.js

```diff
@@ -343,6 +371,20 @@ class UpdateChannelMspModal extends React.Component {
 		return fields;
 	}
 
+	populateMSPAdmins()
```

Comment:
this is close, but not quite what i had in mind. instead of looking at the node ou in the certs in the wallet, look in the config block for this channel, and find the MSP section and its root certs, then compare these certs to the ones in the wallet, if they match, add it the wallet identity's name to the list

you will need to reach back a bit because `UpdateChannelMspModal` doesn't have this data, and it's parent `ChannelMembers` doesn't have it either, but that guy's parent `ChannelDetails` does. so just feed it down as arguments to each component until `UpdateChannelMspModal` can see the data you need

Reply:
@selvaprakash92 this part has not been done